org.osgi.test.assertj.framework/src/test/java/org/osgi/test/assertj/testutil/ConditionAssert.java

```diff
@@ -0,0 +1,57 @@
+package org.osgi.test.assertj.testutil;
```

Comment:
License comment

Reply:
added.

Reply:
All the test code should be under `org.osgi.test.assertj.test.*`.